logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -54,21 +58,26 @@ interface UserRepository {
     suspend fun observeUser(userId: UserId): Flow<User?>
     suspend fun userById(userId: UserId): Either<CoreFailure, OtherUser>
     suspend fun updateSelfUserAvailabilityStatus(status: UserAvailabilityStatus)
+    suspend fun updateOtherUserAvailabilityStatus(userId: UserId, status: UserAvailabilityStatus)
     suspend fun getAllKnownUsersNotInConversation(conversationId: ConversationId): Either<StorageFailure, List<OtherUser>>
+    suspend fun getUsersFromTeam(teamId: TeamId): Either<StorageFailure, List<OtherUser>>
+    suspend fun getTeamRecipients(teamId: TeamId): Either<CoreFailure, List<Recipient>>
```

Comment:
these changes are needed for the future when we'll broadcast our own status to others. I just left it here, cause needed to cut off the work on this task for now, and it doesn't break anything